packages/docs/src/helpers/CodeSandboxExample.ts

```diff
@@ -3,6 +3,10 @@ import { iconsStyles, iconsConfig } from './CodeSandboxIconsHelper'
 import { CodesandboxConfig } from '../types/configTypes'
 // @ts-ignore
 import packageUi from 'vuestic-ui/package.json'
+// @ts-ignore
+import exampleTsconfig from './codesandbox-example-config/example-tsconfig.json'
+// @ts-ignore
+import exampleTsConfigNode from './codesandbox-example-config/example-tsconfig.node.json'
```

Comment:
About this `ts-ignore`, maybe better to declare the *.json modules or enable the `resolveJsonModule` flag in the `packages/docs` typescript configuration?